Description: Temporarily drop openstackdocstheme sphinx extension
  until sphinx>=1.6.2 is available.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: no
Last-Update: 2017-08-24

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -39,7 +39,7 @@
 # They can be extensions coming with Sphinx (named 'sphinx.ext.*')
 # or your custom ones.
 extensions = [
-    'openstackdocstheme',
+    #'openstackdocstheme',
     'sphinx.ext.autodoc',
     'wsmeext.sphinxext',
     'sphinx.ext.coverage',
@@ -115,7 +115,7 @@
 # a list of builtin themes.
 # html_theme_path = ['.']
 # html_theme = '_theme'
-html_theme = 'openstackdocs'
+# html_theme = 'openstackdocs'
 
 # openstackdocstheme options
 repository_name = 'openstack/ceilometer'
